paperqa/agents/models.py

```diff
@@ -85,7 +85,7 @@ def set_docs_name(self, docs_name: str) -> None:
 
 
 class AnswerResponse(BaseModel):
-    answer: Answer
+    answer: PQASession
```

Comment:
Should we rename this to `session` with an `alias` to `answer`?

Reply:
Thanks - missed this one